migrations/global_agent/migration_test.go

```diff
@@ -88,7 +88,7 @@ func TestCases(t *testing.T) {
 			output, n, err := config.ApplyMigrations(input)
 			require.NoError(t, err)
 			require.NotEmpty(t, output)
-			require.Positive(t, n, "expected migration application but none applied")
+			require.NotZero(t, n, "expected migration application but none applied")
```

Comment:
Hmmm that's something different, as e.g. `-1` would error before but not now. However, I'm not sure if `ApplyMigrations` returns an unsigned integer but in any case I would prefer the former as it is more informative and actually what we expect.

Reply:
I tried to workaround false positive finding here (`useless-assert: meaningless assertion`).
Since this is `uint64`, it will never be less than `0`. 

But if you want, I can revert it to the previous state and add `//nolint` with a false-positive description. 
Let me know.

Reply:
I do understand but with `Positive` you don't need to know if the result is signed or not, you do directly understand what we want. For `NotZero` you need to know this is uint and cannot be negative. I also veto that `useless-assert: meaningless assertion` is correct here! This is a bug!

Reply:
Back to `Positive` with `//nolint`. I will try to isolate this and fill the bug.